filebeat/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
could you please include the metadata details for `filebeat.registry.unprocessed_bytes` here?

Reply:
(done)